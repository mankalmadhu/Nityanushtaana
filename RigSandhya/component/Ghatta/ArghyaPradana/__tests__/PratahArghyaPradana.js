import React from 'react';
import ಪ್ರಾತರರ್ಘ್ಯ from '../PratahArghyaPradana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../Gayatri/Trihruti',()=>'ತ್ರಿಹೃತಿ');
jest.mock('../../Gayatri/GayatriMantra',()=>'ಗಾಯತ್ರೀಮ೦ತ್ರ');
jest.mock('../../Sankalpa/Oomapo',()=>'ಓಮಾಪೋ');

test("ಪ್ರಾತರರ್ಘ್ಯ", () => {
   expect(Renderer.create(
    <ಪ್ರಾತರರ್ಘ್ಯ />
  ).toJSON()).toMatchSnapshot();

});