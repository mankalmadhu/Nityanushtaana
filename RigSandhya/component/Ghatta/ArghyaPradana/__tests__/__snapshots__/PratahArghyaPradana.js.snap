// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತರರ್ಘ್ಯ 1`] = `
<View>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    ಅರ್ಘ್ಯ ಪ್ರದಾನ
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    (ಪೂರ್ವಾಭಿಮುಖವಾಗಿ ನಿ೦ತು)
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಪೂರ್ವಾ೦ಗ"
  >
    ತತ್ಸವಿತುರ್ವಿಶ್ವಾಮಿತ್ರಸ್ಸವಿತಾಗಾಯತ್ರೀ
  </ವಿಷಯ>
  <ತ್ರಿಹೃತಿ>
    <ಗಾಯತ್ರೀಮ೦ತ್ರ />
    <ಓಮಾಪೋ />
  </ತ್ರಿಹೃತಿ>
  <ವಿನಿಯೋಗ>
    (೩ ಬಾರಿ)
  </ವಿನಿಯೋಗ>
  <ಕಾಲಾತೀತಅರ್ಘ್ಯ />
  <ವಿನಿಯೋಗ>
    ಓ೦ ಯದ॒ದ್ಯ ಕಚ್ಚ॑ ವೃತ್ರಹನ್ನು॒ದಗಾ᳚ ಅ॒ಭಿ ಸೂ᳚ರ್ಯ । ಸರ್ವ॒೦ ತದಿ᳚೦ದ್ರ ತೇ॒ ವಶೇ᳚ ॥ ಸೂರ್ಯ ಅಹ೦ ಸೂರ್ಯಃ । ಅಸಾವಾದಿತ್ಯೋ ಬ್ರಹ್ಮ ॥
  </ವಿನಿಯೋಗ>
</View>
`;
